package dev.marcal.chatvault.in_out_boundary.output.exceptions

class ChatNotFoundException(message: String? = null, throwable: Throwable? = null): RuntimeException(message, throwable)